app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
```

Comment:
Using a class-level dictionary `people` to store instances of `Person` by name can lead to issues if two people have the same name. Consider using a unique identifier or handling name collisions.